test/base/twilltestcase.py

```diff
@@ -1519,6 +1522,7 @@ def login( self, email='test@bx.psu.edu', password='testuser', username='admin-u
     def logout( self ):
         self.visit_url( "%s/user/logout" % self.url )
         self.check_page_for_string( "You have been logged out" )
+        tc.browser.cj.clear()
```

Comment:
I think that this addition (and the one in `galaxy_logout()`) are causing the Tool Shed test failures. The reason is that clearing the browser cookies will logout both the Galaxy and TS sessions.